lib/msf/ui/console/command_dispatcher/exploit.rb

```diff
@@ -138,6 +139,7 @@ def cmd_exploit(*args)
       return false
     end
 
+    driver.run_single('reload_lib -a') if args[:reload_libs]
```

Comment:
Needs to happen earlier before validation etc occurs, especially if people are changing and reloading the validation logic libraries, so probably line 96